projects/element-ng/toast-notification/si-toast-notification-drawer/si-toast-notification-drawer.component.css

```
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+
+/* Workaround for cdk-overlay not supporting the order of overlays */
+::ng-deep .cdk-global-overlay-wrapper:has(si-toast-notification-drawer) {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)